src/Type/Php/FilterVarDynamicReturnTypeExtension.php

```diff
@@ -28,6 +28,11 @@
 class FilterVarDynamicReturnTypeExtension implements DynamicFunctionReturnTypeExtension
 {
 
+	/**
+	 * All validation filters match 0x100.
+	 */
+	private const VALIDATION_FILTER_BITMASK = 0x100;
```

Comment:
Just curious, do we know if this is intentional, or a coincidence?

Reply:
Looks intentional to me. I went through the constants and all validation filters start with 0x100, and all sanitization filters start with 0x200. FILTER_CALLBACK is 0x400. I figured taking advantage of the pattern was better than iterating through every possible filter that should match.

Reply:
Makes sense, thanks for checking that